controlplane/api/v1alpha1/rke2controlplane_types.go

```diff
@@ -169,6 +169,10 @@ type RKE2ControlPlaneStatus struct {
 
 	// UnavailableReplicas is the number of replicas current attached to this ControlPlane Resource and that are up-to-date with Control Plane config
 	UnavailableReplicas int32 `json:"unavailableReplicas,omitempty"`
+
+	// AvailableServerIPs is the ip address of the initial Control Plane node that is used to register all subsequent nodes
```

Comment:
Comment implies a singular address, where the field is a slice (i.e.implies multiple)

Reply:
Indeed, the comment is out-of-sync! I updated it.